enos/modules/vault_cluster/main.tf

```diff
@@ -37,6 +41,13 @@ locals {
     "pkcs11" = null
   }
   leader = toset(slice(local.instances, 0, 1))
+  netcat_command = {
```

Comment:
I though we created this in globals. Why are we redefining it here?

Reply:
In globals, we have a list of the packages that need to be installed for each distro:
```
distro_packages = {
    amzn2 = ["nc"]
    leap  = ["netcat", "openssl"]
    rhel  = ["nc"]
    sles   = ["nc", "openssl"]
    ubuntu = ["netcat"]
```
We use a separate set of package names here for two reasons:
1) We are getting the name of the distro from `enos_host_info`, which returns slightly different distro names than the ones we use throughout the scenarios (e.g. `opensuse-leap` vs `leap`, `amzn` vs `amzn2`).
2) We include _only_ the netcat CLI command used by the distro, as a single string. It seemed rather fragile to try to pick "the netcat one" from a list of multiple packages in our globals.